app/src/utils/utils.ts

```diff
@@ -81,9 +86,21 @@ export function isValidTwitter(val: string | undefined) {
   return /^[a-zA-Z0-9_]{1,15}$/.test(handle);
 }
 
-// Returns true if the provided address is valid (TODO support ENS)
-export function isValidAddress(val: string | undefined) {
-  return val && isAddress(val);
+// Returns true if the provided logo is valid
+export async function isValidLogo(file: File | undefined) {
+  if (!file) return true;
```

Comment:
shouldn't this be false ? Or are we saying no file is true AKA no error is thrown ?

Reply:
Yep the latter, we default to true otherwise you'd get an "invalid image" error as soon as the form loads

Reply:
it's true, because logo is optional so if no file is uploaded, logo is undefined